### ISSUE TITLE
Assignment data and material handling 

### ISSUE THREAD

**Issue by MarionaRieraGarrofe** [janboone/msc_datascience#30]
Hello, 

I have some questions regarding the assignment data and the material handling. 

While doing the assignment we have assumed that the Altair visualizations would go in the figures folder of jupyterlab (as in class). Once we handle the assignment, should then we also should push the 2 Altair visualizations onto our assignment's github repository? 

Our data is similarly extracted from the Eurostat source, as it was done in class.  We have extracted variable by variable from the API and then we have cleaned each variable before merging it into a big dataset (all of this information has been explained in the assignment).  Then, since we have merged all the clean data in one big cleaned dataset, should we only have to handle the cleaned dataset in our assignment's github repository? Or should we create a new uncleaned dataset and present the not cleaned data in our github repository so you can replicate it? 

When we’ll handle the assignment notebook, all the cells will be run so the outcomes can be seen. However, in case that you intend to replicate the assignment and run the cells in jupyterlab since we have a lot of material (mainly because of the bayesian models) it takes a lot of time for all the cells to be run. Is that a problem? 

To sum up, then in the assignment repository we would have the altair visualizations, the cleaned dataset (since having the not cleaned dataset seems complicated since it is extracted one by one from Eurostat database), and the assignment notebook. 

Is this okay? 

Thanks

**Comment by janboone** [comment]
> While doing the assignment we have assumed that the Altair visualizations would go in the figures folder of jupyterlab (as in class). Once we handle the assignment, should then we also should push the 2 Altair visualizations onto our assignment's github repository?

Indeed, it is useful for us if everything is pushed on the github repos.

> Our data is similarly extracted from the Eurostat source, as it was done in class. We have extracted variable by variable from the API and then we have cleaned each variable before merging it into a big dataset (all of this information has been explained in the assignment). Then, since we have merged all the clean data in one big cleaned dataset, should we only have to handle the cleaned dataset in our assignment's github repository? Or should we create a new uncleaned dataset and present the not cleaned data in our github repository so you can replicate it?

As long as we have the *code* to replicate your results, this is fine. We do not need the intermediate outputs (uncleaned datasets).

> When we’ll handle the assignment notebook, all the cells will be run so the outcomes can be seen. However, in case that you intend to replicate the assignment and run the cells in jupyterlab since we have a lot of material (mainly because of the bayesian models) it takes a lot of time for all the cells to be run. Is that a problem?

This is not a problem.

> To sum up, then in the assignment repository we would have the altair visualizations, the cleaned dataset (since having the not cleaned dataset seems complicated since it is extracted one by one from Eurostat database), and the assignment notebook.

Indeed, this is fine.
